README.md

```diff
@@ -1,13 +1,13 @@
 # Kong Developer Portal CLI
 [![License](https://img.shields.io/github/license/kong/kong-portal-cli.svg)][cli-license]
 
-The Kong Developer Portal CLI is used to manage your Developer Portals from the 
+The Kong Developer Portal CLI is used to manage your Developer Portals from the
 command line. It is built using [clipanion][clipanion].
 
 ## Overview
 
-This is the next generation TypeScript based Developer Portal CLI. The goals of 
-this project were to make higher quality CLI tool over the initial sync script.
+This is the next generation TypeScript based Developer Portal CLI. The goal of
+this project was to make higher quality CLI tool over the initial sync script.
```

Comment:
"this project is to make a higher quality CLI tool over the initial sync script."

Reply:
I like that